packages/core/test/tags.test.ts

```diff
@@ -0,0 +1,54 @@
+import { createMachine } from '../src';
+
+describe('tags', () => {
```

Comment:
we need a test like this:
1. two parallel regions with the same tag
2. exit one of them
3. the expected result is that we should still have that tag available to us

Reply:
Done